### PR TITLE
 Update spanish translation

### PR DESCRIPTION
**Pull request by estiloinfo**
Added missing strings

### CODE REVIEW

**Comment by luixxiul** [comment]
ref: https://github.com/jitsi/jitsi-meet/issues/6847

**Comment by estiloinfo** [comment]
I sign CLA

File: lang/main-es.json [hunk 1]
```diff
@@ -32,10 +47,10 @@
         "confirmAddLink": "¿Quiere añadir un enlace de Jitsi a este evento?",
         "error": {
             "appConfiguration": "La integración del calendario no está configurada correctamente",
-            "generic": "Se ha producido un error. Compruebe la configuración del calendario o pruebe a recargarlo",
+            "generic": "Se ha producido un error. Compruebe la configuración del calendario o pruebe cargarlo nuevamente.",
             "notSignedIn": "Se ha producido un error de autenticación para ver los eventos del calendario. Compruebe la configuración del calendario e intente iniciar sesión de nuevo"
         },
-        "join": "Unirse",
+        "join": "Unir",
```
Comment (saghul) [inline lang/main-es.json:53]:
```suggestion
        "join": "Unirse",
```

File: lang/main-es.json [hunk 2]
```diff
@@ -114,7 +130,7 @@
         "transport_plural": "Transportes:"
     },
     "dateUtils": {
-        "earlier": "Anterior",
+        "earlier": "Temprano",
```
Comment (saghul) [inline lang/main-es.json:133]:
```suggestion
        "earlier": "Anterior",
```

File: lang/main-es.json [hunk 3]
```diff
@@ -175,6 +195,10 @@
         "dismiss": "Descartar",
         "displayNameRequired": "¡Hola! ¿Cuál es tu nombre?",
         "done": "Listo",
+        "e2eeDescription": "<p>El cifrado Punto-a-Punto es actualmente <strong>EXPERIMENTAL</strong>. Por favor lea<a href='https://jitsi.org/blog/e2ee/' target='_blank'>este artículo</a> para más detalles.</p><br/><p>Tenga en cuenta que activar el cifrado punto-a-punto puede deshabilitar servicios en el servidor como: grabación, transmisión en vivo y participación telefónica. Sin embargo tenga en cuenta que esta reunion solo funcionará con personas que se unan usando un navegador.</p>",
```
Comment (saghul) [inline lang/main-es.json:198]:
```suggestion
        "e2eeDescription": "<p>El cifrado Extremo-a-Extremo es actualmente <strong>EXPERIMENTAL</strong>. Por favor lea<a href='https://jitsi.org/blog/e2ee/' target='_blank'>este artículo</a> para más detalles.</p><br/><p>Tenga en cuenta que activar el cifrado extremo-a-extremo puede deshabilitar servicios en el servidor como: grabación, transmisión en vivo y participación telefónica. Sin embargo tenga en cuenta que esta reunion solo funcionará con personas que se unan usando un navegador.</p>",
```

File: lang/main-es.json [hunk 4]
```diff
@@ -175,6 +195,10 @@
         "dismiss": "Descartar",
         "displayNameRequired": "¡Hola! ¿Cuál es tu nombre?",
         "done": "Listo",
+        "e2eeDescription": "<p>El cifrado Extremo-a-Extremo es actualmente <strong>EXPERIMENTAL</strong>. Por favor lea<a href='https://jitsi.org/blog/e2ee/' target='_blank'>este artículo</a> para más detalles.</p><br/><p>Tenga en cuenta que activar el cifrado extremo-a-extremo puede deshabilitar servicios en el servidor como: grabación, transmisión en vivo y participación telefónica. Sin embargo tenga en cuenta que esta reunion solo funcionará con personas que se unan usando un navegador.</p>",
+        "e2eeLabel": "Clave",
+        "e2eeTitle": "Cifrado Punto-a-Punto",
```
Comment (saghul) [inline lang/main-es.json:200]:
```suggestion
        "e2eeTitle": "Cifrado Extremo-a-Extremo",
```

File: lang/main-es.json [hunk 5]
```diff
@@ -175,6 +195,10 @@
         "dismiss": "Descartar",
         "displayNameRequired": "¡Hola! ¿Cuál es tu nombre?",
         "done": "Listo",
+        "e2eeDescription": "<p>El cifrado Extremo-a-Extremo es actualmente <strong>EXPERIMENTAL</strong>. Por favor lea<a href='https://jitsi.org/blog/e2ee/' target='_blank'>este artículo</a> para más detalles.</p><br/><p>Tenga en cuenta que activar el cifrado extremo-a-extremo puede deshabilitar servicios en el servidor como: grabación, transmisión en vivo y participación telefónica. Sin embargo tenga en cuenta que esta reunion solo funcionará con personas que se unan usando un navegador.</p>",
+        "e2eeLabel": "Clave",
+        "e2eeTitle": "Cifrado Extremo-a-Extremo",
+        "e2eeWarning": "<br /><p><strong>ATENCION:</strong> No todos los participantes de esta reunión soportan cifrado Punto-a-Punto. Si usted habilita el cifrado ellos no podrán verlo ni oirlo.</p>",
```
Comment (saghul) [inline lang/main-es.json:201]:
```suggestion
        "e2eeWarning": "<br /><p><strong>ATENCION:</strong> No todos los participantes de esta reunión soportan cifrado Extremo-a-Extremo. Si usted habilita el cifrado ellos no podrán verlo ni oirlo.</p>",
```

File: lang/main-es.json [hunk 6]
```diff
@@ -290,6 +315,9 @@
     "documentSharing": {
         "title": "Compartir documento"
     },
+    "e2ee": {
+        "labelToolTip": "La comunicación de Audio y Video en esta reunión está cifrada Punto-a-Punto"
```
Comment (saghul) [inline lang/main-es.json:319]:
```suggestion
        "labelToolTip": "La comunicación de Audio y Video en esta reunión está cifrada Extremo-a-Extremo"
```

File: lang/main-es.json [hunk 7]
```diff
@@ -587,6 +662,7 @@
             "chat": "Alternar ventana de chat",
             "document": "Alternar documento compartido",
             "download": "Descarga nuestras aplicaciones",
+            "e2ee": "Cifrado Punto-a-Punto",
```
Comment (saghul) [inline lang/main-es.json:665]:
```suggestion
            "e2ee": "Cifrado Extremo-a-Extremo",
```

File: lang/main-es.json [hunk 8]
```diff
@@ -627,6 +707,7 @@
         "documentClose": "Cerrar documento compartido",
         "documentOpen": "Abrir documento compartido",
         "download": "Descarga nuestras aplicaciones",
+        "e2ee": "Cifrado Punto-a-Punto",
```
Comment (saghul) [inline lang/main-es.json:710]:
```suggestion
        "e2ee": "Cifrado Extremo-a-Extremo",
```

File: lang/main-es.json [hunk 9]
```diff
@@ -613,6 +692,7 @@
             "speakerStats": "Alternar estadísticas del orador",
             "tileView": "Alternar vista de mosaico",
             "toggleCamera": "Alternar cámara",
+            "toggleFilmstrip": "Arternar a película",
```
Comment (saghul) [inline lang/main-es.json:695]:
```suggestion
            "toggleFilmstrip": "Alternar mosaicos",
```

File: lang/main-es.json [hunk 10]
```diff
@@ -603,7 +679,10 @@
             "profile": "Editar tu perfil",
             "raiseHand": "Levantar / Bajar tu mano",
             "recording": "Activar grabación",
+            "raiseHand": "Alternar levantar la mano",
```
Comment (saghul) [inline lang/main-es.json:682]:
```suggestion
```

File: lang/main-es.json [hunk 11]
```diff
@@ -603,7 +679,9 @@
             "profile": "Editar tu perfil",
             "raiseHand": "Levantar / Bajar tu mano",
             "recording": "Activar grabación",
+            "recording": "Alternar grabación",
```
Comment (saghul) [inline lang/main-es.json:682]:
```suggestion
```

File: lang/main-esUS.json [hunk 12]
```diff
@@ -175,6 +194,10 @@
         "dismiss": "Descartar",
         "displayNameRequired": "¡Hola! ¿Cuál es tu nombre?",
         "done": "Listo",
+        "e2eeDescription": "<p>El cifrado Punto-a-Punto es actualmente <strong>EXPERIMENTAL</strong>. Por favor lea<a href='https://jitsi.org/blog/e2ee/' target='_blank'>este artículo</a> para más detalles.</p><br/><p>Tenga en cuenta que activar el cifrado punto-a-punto puede deshabilitar servicios en el servidor como: grabación, transmisión en vivo y participación telefónica. Sin embargo tenga en cuenta que esta reunion solo funcionará con personas que se unan usando un navegador.</p>",
```
Comment (saghul) [inline lang/main-esUS.json:197]:
```suggestion
        "e2eeDescription": "<p>El cifrado Extremo-a-Extremo es actualmente <strong>EXPERIMENTAL</strong>. Por favor lea<a href='https://jitsi.org/blog/e2ee/' target='_blank'>este artículo</a> para más detalles.</p><br/><p>Tenga en cuenta que activar el cifrado extremo-a-extremo puede deshabilitar servicios en el servidor como: grabación, transmisión en vivo y participación telefónica. Sin embargo tenga en cuenta que esta reunion solo funcionará con personas que se unan usando un navegador.</p>",
```

File: lang/main-esUS.json [hunk 13]
```diff
@@ -175,6 +194,10 @@
         "dismiss": "Descartar",
         "displayNameRequired": "¡Hola! ¿Cuál es tu nombre?",
         "done": "Listo",
+        "e2eeDescription": "<p>El cifrado Extremo-a-Extremo es actualmente <strong>EXPERIMENTAL</strong>. Por favor lea<a href='https://jitsi.org/blog/e2ee/' target='_blank'>este artículo</a> para más detalles.</p><br/><p>Tenga en cuenta que activar el cifrado extremo-a-extremo puede deshabilitar servicios en el servidor como: grabación, transmisión en vivo y participación telefónica. Sin embargo tenga en cuenta que esta reunion solo funcionará con personas que se unan usando un navegador.</p>",
+        "e2eeLabel": "Clave",
+        "e2eeTitle": "Cifrado Punto-a-Punto",
```
Comment (saghul) [inline lang/main-esUS.json:199]:
```suggestion
        "e2eeTitle": "Cifrado Extremo-a-Exremo",
```

File: lang/main-esUS.json [hunk 14]
```diff
@@ -1,21 +1,36 @@
...
+        "linkCopied": "Link copiado al portapapeles",
+        "loading": "Buscando por contacto y número telefónico",
+        "loadingNumber": "Validando el número telefónico",
+        "loadingPeople": "Buscando contactos a invitar",        
         "noResults": "No se encontraron coincidencias",
         "noValidNumbers": "Por favor ingrese un número de teléfono",
+        "outlookEmail": "Correo de Outlook",
         "searchNumbers": "Agregar números telefónicos",
-        "searchPeople": "Búsqueda de personas",
+        "searchPeople": "Búscar personas",
```
Comment (saghul) [inline lang/main-esUS.json:26]:
```suggestion
        "searchPeople": "Buscar personas",
```

File: lang/main-esUS.json [hunk 15]
```diff
@@ -175,6 +194,10 @@
         "dismiss": "Descartar",
         "displayNameRequired": "¡Hola! ¿Cuál es tu nombre?",
         "done": "Listo",
+        "e2eeDescription": "<p>El cifrado Extremo-a-Extremo es actualmente <strong>EXPERIMENTAL</strong>. Por favor lea<a href='https://jitsi.org/blog/e2ee/' target='_blank'>este artículo</a> para más detalles.</p><br/><p>Tenga en cuenta que activar el cifrado extremo-a-extremo puede deshabilitar servicios en el servidor como: grabación, transmisión en vivo y participación telefónica. Sin embargo tenga en cuenta que esta reunion solo funcionará con personas que se unan usando un navegador.</p>",
+        "e2eeLabel": "Clave",
+        "e2eeTitle": "Cifrado Extremo-a-Exremo",
+        "e2eeWarning": "<br /><p><strong>ATENCION:</strong> No todos los participantes de esta reunión soportan cifrado Punto-a-Punto. Si usted habilita el cifrado ellos no podrán verlo ni oirlo.</p>",
```
Comment (saghul) [inline lang/main-esUS.json:200]:
```suggestion
        "e2eeWarning": "<br /><p><strong>ATENCION:</strong> No todos los participantes de esta reunión soportan cifrado Extremo-a-Extremo. Si usted habilita el cifrado ellos no podrán verlo ni oirlo.</p>",
```

File: lang/main-esUS.json [hunk 16]
```diff
@@ -244,6 +267,7 @@
         "reservationError": "Error del sistema de reservaciones",
         "reservationErrorMsg": "Código de error: {{code}}, mensaje: {{msg}}",
         "retry": "Volver a intentar",
+        "screenSharingAudio": "Share audio",
```
Comment (saghul) [inline lang/main-esUS.json:270]:
```suggestion
        "screenSharingAudio": "Compartir audio",
```

File: lang/main-esUS.json [hunk 17]
```diff
@@ -289,6 +313,9 @@
     "documentSharing": {
         "title": "Documento Compartido"
     },
+    "e2ee": {
+        "labelToolTip": "La comunicación de Audio y Video en esta reunión está cifrada Punto-a-Punto"
```
Comment (saghul) [inline lang/main-esUS.json:317]:
```suggestion
        "labelToolTip": "La comunicación de Audio y Video en esta reunión está cifrada Extremo-a-Extremo"
```

File: lang/main-esUS.json [hunk 18]
```diff
@@ -586,6 +661,7 @@
             "chat": "Alternar ventana de chat",
             "document": "Alternar documento compartido",
             "download": "Descargar nuestras aplicaciones",
+            "e2ee": "Cifrado Punto-a-Punto",
```
Comment (saghul) [inline lang/main-esUS.json:664]:
```suggestion
            "e2ee": "Cifrado Extremo-a-Extremo",
```

File: lang/main-esUS.json [hunk 19]
```diff
@@ -614,6 +691,7 @@
             "speakerStats": "Alternar estadísticas del altavoz",
             "tileView": "Alternar vista de mosaico",
             "toggleCamera": "Alternar cámara",
+            "toggleFilmstrip": "Arternar a película",
```
Comment (saghul) [inline lang/main-esUS.json:694]:
```suggestion
            "toggleFilmstrip": "Alternar mosaicos",
```

File: lang/main-esUS.json [hunk 20]
```diff
@@ -628,6 +706,7 @@
         "documentClose": "Cerrar documento compartido",
         "documentOpen": "Abrir documento compartido",
         "download": "Descargar nuestras aplicaciones",
+        "e2ee": "Cifrado Punto-a-Punto",
```
Comment (saghul) [inline lang/main-esUS.json:709]:
```suggestion
        "e2ee": "Cifrado Extremo-a-Extremo",
```

**Pull request merged**